inc/datahandlers/post.php

```diff
@@ -1255,6 +1249,13 @@ function insert_post()
 			$thread_update = array('replies' => '+1');
 			update_forum_counters($post['fid'], array("deletedposts" => "+1"));
 		}
+		
+		// Update forum count
+		if($visible == 1) {
+			update_last_post($post['tid']);
+			update_forum_counters($post['fid'], array("posts" => "+1"));
```

Comment:
Same here... The forum counters shouldn't be changes if the thread isn't visible. Updating the thread's last post is correct though.